app/services/audit/sdr_warc_lister.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Audit
+  # Enumberable that returns WARC filenames for a collection from SDR.
+  class SdrWarcLister
+    include Enumerable
+    def initialize(collection_druid:)
+      @collection_druid = collection_druid
+    end
+
+    def each(&block)
+      if block_given?
+        member_druids.each do |member_druid|
+          files_for(member_druid).each { |file| block.call(file) }
+        end
+      else
+        to_enum(:each)
+      end
+    end
+
+    private
+
+    attr_reader :collection_druid
+
+    def member_druids
+      Dor::Services::Client.object(collection_druid).members.map(&:externalIdentifier)
+    end
+
+    def files_for(member_druid)
+      cocina_obj = Dor::Services::Client.object(member_druid).find
+      return [] unless cocina_obj.type == Cocina::Models::ObjectType.webarchive_binary
```

Comment:
Although crawls accessioned via WRA were registered as webarchive-binary, I think the type was previously getting overwritten through the metadata steps in wasCrawlPreassembly. So this would result in objects like [this example]( https://argo.stanford.edu/view/druid:fv721mj9383 ) not getting included.

Reply:
Thanks for the catch. Switching to returning if a seed.